presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -189,6 +189,23 @@ public long getTotalMemoryReservation()
     }
 
     @Override
+    public long getTotalRevocableMemoryReservation()
+    {
+        // acquire reference to outputStage before checking finalQueryInfo, because
```

Comment:
to be honest I have no idea - I have copied paste `getTotalMemoryReservation` method